sra-schema/EGA_SRA_Metadata.csv

```diff
@@ -0,0 +1,89 @@
+Table 1
+Agent that provides to QBIC;Existing property in OpenBIS;New property in OpenBIS;OpenBIS object type;Metadata Level;Specific metadata;SRA Structured comment name;SRA Item;EGA metadata object;EGA item;EGA label;Definition;Expected value;OpenBIS type;json type;Value syntax;Example;Section;migs_eu;migs_ba;migs_pl;migs_vi;migs_org;me;mimarks_s;mimarks_c;misag;mimag;miuvig;ega;Plant;Virus;SarsCov;PathogenEnv;PathogenCl;Microbe;Metagenome;Invertebrate;Human;BetaLactamase

```

Comment:
This `.csv` file is actually separated by `;`, could it be edited so that it is trully comma `,` separated? Also without extra lines or anything, so that it can be read directly as csv